scss/detail.scss

```diff
@@ -34,133 +26,344 @@ $container-width: 900px;
+    padding: 24px;
+    width: 615px;
+  
+    .header-detail {
+      align-items: center;
+      border-bottom: 4px solid $green-lemon;
+      display: flex;
+      padding-bottom: 12px;
+      width: 100%;
+      
+      .title-book {
+        color: $black-default;
+        font-weight: 700;
+        margin-left: 8px;
+        font-size:  32px;
+        line-height: 32px;
+      }
+      
+      .title-gender {
+        color: $grey;
+        font-weight: 700;
+        padding-left: 8px;
+        font-size: 24px;
+        line-height: 24px;
+      }
+    }
+    
+    .detail-form {
+      margin: 36px 0 0 8px;
+    
+      .detail-item {
+        color: $black-default;
+        font-weight: 700;
+        font-size: 20px;
+        line-height: 20px;
+      }
+      
+      .detail-description {
+        color: $grey;
+        margin-left: 8px;
+        font-size: 20px;
+        line-height: 20px;
+      }
+    }
+  }
 }
 
 @media(max-width: $breakpoint) {
-  .back-arrow {
-    width: calc(#{$container-width} - 564px);
-    height: 20px;
-    margin-top: 33px;
-    justify-content: center;
+  .container{
     align-items: center;
+    background-color: $white-primary;
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    height: calc(#{$total-height} - 101px);
+    width: calc(#{$total-width} - 990px);
+    
+    .back-arrow {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      max-width: 1024px;
+      width: auto;
+      height: 20px;
+      margin-top: 33px;
+      cursor: pointer;
+    }
+    
+    .container-book {
+      flex-direction: column;
+      align-items: center;
+      height: 570px;
+      width: 336px;
+      
+      .container-img {
+        height: 337px;
+        padding: 38px 48px 0 48px;
+        width: 336px;
+    
+        .badge {
+          height: calc($badge-height + 25%);
+          padding: 18px;
+          width: calc($badge-width + -30%);
+          right: 10px;
+          top: 8px;
+        }
+      }
+      
+      .container-detail {
+        height: 233px;
+        padding: 21px;
+        width: 336px;
+        
+        .header-detail {
+          .title-book{
+            font-size: 24px;
+            line-height: 24px;
+          }
+      
+          .title-gender {
+            font-size: 16px;
+            line-height: 16px;
+          }
+        }
+      
+        .detail-form {
+          margin-top: 18px;
+      
+          .detail-item {
+            font-size: 14px;
+            line-height: 14px;
+          }
+      
+          .detail-description {
+            font-size: 14px;
+            line-height: 14px;
+          }
+        }
+      }
+    }
   }
+}
 
-  .container-book {
-    flex-direction: column;
-    align-items: center;
-    height: calc(#{$container-height} + 151px);
-    width: calc(#{$container-width} - 564px);
+@media(max-width: 853px) {
+  .container {
+    height: 768px;
+    width: 1024px;
```

Comment:
The breakpoint max-width is lesser than the container width so we can only see up to 853px of width. Is it necessary to define 1024px of width? :thinking: